sass/type/_helpers.scss

```diff
@@ -147,19 +144,15 @@ $_a_FONT-FORMATS: (
 ///   SVG or EOT-iefix suffix to be added to font url.
 /// @return {list} -
 ///   CSS-ready `url() format()` combination for font-face `src` output.
-@function _a_font-src(
-  $path,
-  $format
-) {
-  $format-name: map-get($_a_FONT-FORMATS, $format);
+@function _a_font-src($path, $format) {
+  $format-name: map.get($FONT-FORMATS, $format);
   $format-name: if($format-name, format($format-name), '');
 
   // stylelint-disable-next-line function-url-quotes
-  $path: _a_font-url($path, $format, $iefix: true);
+  $path: font-url($path, $format, $iefix: true);
   @return font-url($path) $format-name;
```

Comment:
@mirisuzanne As you know, the second `font-url` function here is defined in the `fonts` module. if I use that module here and try to call it with `fonts.font-url` I get a module loop error:

```
Error: Module loop: this module is already being loaded.
  ┌──> sass/type/_helpers.scss
5 │ @use './fonts';
  │ ^^^^^^^^^^^^^^ new load
  ╵
  ┌──> test/type/_fonts.scss
3 │ @use '../../../sass/type/fonts';
  │ ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ original load
```
Would the best way to go about this be to relocate/rename the `font-url` function in `fonts`?

Reply:
For sure we should rename one or both of the font-url functions, to clarify what they do. But then, yeah, we can't have both modules relying on the other - so _something_ has to relocate. Maybe it works to move the `fonts` one into helpers? Since that's the public one, it's also the one that should get priority for keeping the name.

Reply:
@mirisuzanne 
I ended up moving the `fonts` version into `helpers` and renamed the `helpers` version to `format-url`.

All `type` tests are passing now except for one in `fonts`.

 ```
Import Webfonts [mixin]
         Font-face imports for all local fonts & variants:

```

If I remove the two FranklinGothic `@font-face` blocks, the test passes. I think this means that the test is looking for those two fonts in the global `config.$fonts map`. 

When I add FranklinGothic to the `test-data` font map, it makes the first test in type/config fail
![image](https://user-images.githubusercontent.com/41588129/142269732-dadff4e4-75b8-4a73-b081-29a9ded10713.png)

I think that tests assumes that Franklin isn't in the global fonts map, right? 

Should I just remove to two `@font-face` blocks for `FranklinGothic` in the `Import Webfonts [mixin]` test? 

We can schedule a time to walk through this if this is hard to follow :)